src/components/DeploymentQuickRun.vue

```diff
@@ -0,0 +1,42 @@
+<template>
```

Comment:
Same comment as above - can we rename this to something like `DeploymentQuickRunOverflowMenuItem`?

Reply:
+1